### ISSUE TITLE
WinDivert-2.1.0-A not working in Win10

### ISSUE THREAD

**Issue by pbix** [basil00/Divert#203]
I am unable to make WinDivert run on my Windows 10 PC.

Things I have done:
1) Downloaded the 2.1.0-A zip file.
2) Opened a command prompt as Administrator
3) Execute Test.exe

Error I get is
"error: failed to open WinDivert handle (err = 3)"

Screen shot below.

![WinDivert Failure](https://user-images.githubusercontent.com/289729/62838433-06407480-bc4a-11e9-9fc6-2ff0040b5110.jpg)

**Comment by basil00** [comment]
I tried from a fresh download and it works for me.  I suspect the problem is that there is an old WinDivert service that is pointing to a path that no longer exists.  Try the following as Administrator:

    sc stop windivert
    sc delete windivert

If that does not work, try the following from the `WinDivert-2.1.0-A\x64\` directory:

    windivertctl.exe uninstall

If that does not work, try rebooting.

If rebooting does not work and as a last resort, use `regedit` to manually delete the `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\WinDivert` entry.

**Comment by pbix** [comment]
After my report I tried version 1.4.3 and found that worked OK.  I also found that v2.0 was blocked by my anti-virus software Sophos.

I followed your instruction #1 and found success.  As shown in the screen shot below it was not running but apparently it was installed.   After that the test.exe ran and produced many colorful lines of output :-).   Thanks so much for your help on this.

Looks like a very nice package with nice capability and great documentation.  Thanks again for developing and supporting it, being an OpenSource developer myself I know what kind of work goes into something like this.  Will see what I can do with it.

![WinDivert Service](https://user-images.githubusercontent.com/289729/62840657-457fbd00-bc6c-11e9-86f9-d49837d00508.jpg)
